src/index.js

```diff
@@ -847,30 +864,46 @@ class MiniCssExtractPlugin {
               return null;
             }
 
+            const attributes = {
+              href: `${RuntimeGlobals.publicPath} + ${RuntimeGlobals.require}.miniCssF(chunkId)`,
+              rel: JSON.stringify('stylesheet'),
+              onload: 'onLinkComplete',
+              onerror: 'onLinkComplete',
+            };
```

Comment:
> I think we can write warning in docs about some attributes, they are still internal and should not change, it will simplify logic

@alexander-akait I started rebasing and changing the code based on your comments. Are you suggesting that we remove all properties (i.e. attributes that cannot be set through setAttribute) from the above attributes object?

Otherwise I suggest that we change the below declaration to
```js
// Link properties cannot be set using setAttribute, but must instead be set through
// link.property = value. We maintain this list of known properties, so that we can
// combine attributes of both types in the above object.
const linkProperties = ['href', 'rel', 'type', 'onload', 'onerror'];
```

What do you think?

Reply:
I think docs about these attributes will be enough, they are internal and rewriting them is not safe

Reply:
Yes :+1:

Reply:
解决了吗？目前也是这个sri问题，需要这个功能